testing/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -53,7 +53,7 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
      * Change this whenever you add or remove subprojects for distribution-packaged plugins (lib/plugins).
      */
     int getPackagedPluginsJarCount() {
-        76
+        78
```

Comment:
🤔  The subpprojects we extracted do not look like plugins. Should the core lib jar count (above) be incremented instead?

Reply:
The produced jars do end up under `lib/plugins` 🤷

Reply:
Maybe we should take active steps towards removing the distinction in the tests, as recently [discussed](https://gradle.slack.com/archives/C03A0MT6S/p1715857946933779).

Reply:
Sure, however, this is the current structure this PR needs to comply with.